app/views/sales/new.html.erb

```diff
@@ -0,0 +1,13 @@
+<section>
+    <h2><%= form_for [@vendor, @sale] do |f| %>
+      <p>
+        <%= f.label :amount %>
+        <%= f.number_field :amount %>
+        <%= f.hidden_field(:vendor_id, :value => @vendor.id) %> </p>
+      <p>
+        <%= select_tag "sale[product_id]", options_from_collection_for_select(@product, "id", "name") %>
+      </p>
+
+      <%= f.submit 'Create Sale' %>
+    <% end %></h2>
```

Comment:
put the `<h2>` tags on separate lines from the elements around them, since they are encasing a whole batch of things. (Like your `<p>` tags on lines 7/9)